apps/framework-cli/src/framework/python/generate.rs

```diff
@@ -58,6 +58,9 @@ fn map_column_type_to_python(
 ) -> String {
     match column_type {
         ColumnType::String => "str".to_string(),
+        ColumnType::FixedString { length } => {
+            format!("Annotated[bytes, FixedString({})]", length)
```

Comment:
both `FixedString` and `String` in ClickHouse can store arbitrary bytes. If we are saying CH `FixedString` has to correspond to Python `bytes`, then the same should apply to CH `String` as well.